docs/Build/Linux.md

```diff
@@ -23,83 +23,112 @@ dnf install "pkgconfig(jack)" alsa-lib-devel git help2man
 apt install qjackctl
+apt install librtaudio-dev # if building with RtAudio
 ```
 
-Clone the git repo and run `./build` in the src directory or use QtCreator to compile
+Clone the git repo with submodules and run `./build install` in the project
+directory or use QtCreator to compile.
 
-### Other Linux distributions
+### Building and Installation instructions
 
-Install the dependencies listed above. Clone the git repo and run `./build` in the src directory or use QtCreator to compile.
-
-## Build
-You can compile using the build script or QtCreator.
+For other Linux distributions, install the dependencies listed above. Clone the
+git repo and run `./build install` (or `./build` to skip installation) in the
+project directory, or use QtCreator to compile.
 
 To clone the repo in the Terminal:
-$ git clone https://github.com/jacktrip/jacktrip.git
+```sh
+$ git clone --recurse-submodules https://github.com/jacktrip/jacktrip.git
+```
+Note that we need `--recurse-submodules` to also get the submodules!
 
-To compile using the build script:
+Next, navigate to the cloned repository:
 ```sh
 $ cd jacktrip
+```
+
+JackTrip provides a build script designed to simplify the build process and
+providing a number of options. To list them, run:
+```sh
+$ ./build -h
+```
+This should print:
+```sh
+usage:
+ ./build [noclean nojack rtaudio nogui static install [-config static]]
+
+ options:
+ noclean - do not run "make clean" first
+ nojack - build without jack
+ rtaudio - build with rtaudio
+ nogui - build without the gui
+ static - build with static libraries
+ weakjack - build with weak linking of jack libraries
+ install - install jacktrip in system location (uses sudo)
+```
+
+To just compile JackTrip using the build script, run:
+```sh
 $ ./build
 $ cd builddir
 $ ls
 ```
 
-You should see a `qjacktrip` executable and a `jacktrip` symlink in this folder.
+You should see a `jacktrip` in this folder.
+
+To compile **and install** using the build script, run:
+```sh
+$ ./build install
+# enter your password when prompted
+```
 
-If the build script doesn't work on a different Linux flavor, try building
-the Makefiles yourself. You'd need qmake. Then you can build by:
+If the build script doesn't work, try running qmake directly. You'd need to have
+qmake in your `$PATH`. Then you can build with:
 
 ```sh
-$ qmake jacktrip.pro
+$ mkdir builddir
+$ cd builddir
+$ qmake ../jacktrip.pro
 $ make release
+$ sudo make release-install # to install JackTrip system-wide
 ```
 
+You can pass configuration options to qmake with `-config <option>`, e.g. `qmake
+-config nogui ../jacktrip.pro`. Relevant options include:
+- `-config rtaudio` - build with rtaudio support
+- `-config nogui` - build without the GUI
+- `-config static` - build with static libraries
+- `-config weakjack` - build using weak linking of jack
+- `-config nojack` - build without jack support
+
 To build using QtCreator:
   * Open jacktrip.pro using QtCreator
```

Comment:
Not your change. But this doesn't print correctly. Maybe you want to fix this too.

Reply:
Thanks for catching. I think my list before renders wrong too - MkDocs seems to require an empty line before a list (my local Markdown preview doesn't)